love/src/components/ScriptQueue/ConfigPanel/ConfigPanel.jsx

```diff
@@ -668,7 +717,7 @@ export default class ConfigPanel extends Component {
     // RJSF variables
     let rjsfSchema;
     try {
-      rjsfSchema = yamlSchema ? yaml.load(yamlSchema) : {};
+      rjsfSchema = yamlSchema ? yaml.load(yamlSchema) ?? {} : {};
```

Comment:
Giving a second, look, as we just must ensure `yaml.load(yamlSchema)` is different to `null` or `undefined`, we could change this as follows:
```suggestion
      rjsfSchema = yaml.load(yamlSchema) ?? {};
```
This should work as if `yamlSchema` is `null` or `undefined`, then `yaml.load(yamlSchema)` will be falsy leading to the `?? {}`.